### PR TITLE
Lighter vae demo

### PR DESCRIPTION
**Pull request by psc-g**
This PR creates buttons to allow generating MusicVAE samples only on request. Otherwise, the page is _very_ heavy (it rendered my Lenovo linux laptop useless for a few minutes).

### CODE REVIEW

**Comment by psc-g** [comment]
Signed the CLA.

**Comment by psc-g** [comment]
I am covered by the Google Corporate CLA.

File: music/demos/music_vae.ts [hunk 1]
```diff
@@ -458,11 +458,120 @@ async function runMultitrackChords() {
...
+}
+
+async function generateMultitrackChordsButton() {
+  const multitrackChordsButton = document.createElement('button');
+  multitrackChordsButton.textContent = 'Generate Chord-Conditioned Multitrack';
+  multitrackChordsButton.addEventListener('click', () => {
+    runMultitrackChords();
+    multitrackChordsButton.disabled = true;
+  });
+  const multitrackChordsiv = document.getElementById(
```
Comment (iansimon) [inline music/demos/music_vae.ts]: multitrackChordsDiv (though it doesn't really matter because #87 is going to move this to another file anyway)

**Comment by adarob** [comment]
Pablo, you should figure out why we keep getting those CLA messages. If you
have things set up properly, they won't appear.

**Comment by psc-g** [comment]
joined google github for cla

**Pull request merged**